tests/Providers/Reviewsio/Request/Service/EmailInviteRequestTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace PodPoint\Reviews\Tests\Providers\ReviewsIo\Request\Service;
+
+use PodPoint\Reviews\Providers\ReviewsIo\Request\Service\EmailInviteRequest;
+use PodPoint\Reviews\Tests\TestCase;
+
+class EmailInviteRequestTest extends TestCase
+{
+    /**
+     * Test construct to make sure properties are set.
+     *
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testConstruct()
+    {
+        $mockedApiClient = $this->getMockedApiClient();
+        $request = new EmailInviteRequest($mockedApiClient, [
+            'name' => 'Customer Name',
+            'email' => 'customer@email.com',
+            'order_id' => 'order-id-123',
+            'store' => 'store-id-321',
+        ]);
+
+        $this->assertEquals($mockedApiClient, $request->getHttpClient());
+        $this->assertEquals([
+            'name' => 'Customer Name',
+            'email' => 'customer@email.com',
+            'order_id' => 'order-id-123',
+            'store' => 'store-id-321',
+        ], $request->getOptions());
+    }
+
+    /**
+     *
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testRequiredFields()
+    {
+        $mockedApiClient = $this->getMockedApiClient();
+        $request = new EmailInviteRequest($mockedApiClient, [
+            'name' => 'Customer Name',
+            'email' => 'customer@email.com',
+            'order_id' => 'order-id-123',
+            'store' => 'store-id-321',
+        ]);
+
+        $this->assertEquals([
+            'name',
+            'email',
+            'order_id',
+            'store',
+        ], $request->requiredFields());
+    }
+
+    /**
+     * Making sure the Request instance is build as expected.
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
```

Comment:
I think typically we put space between description and @params @throws @return etc.